week-2/implement/cases.js

```diff
@@ -15,3 +15,11 @@
 
 // Come up with a clear, simple name for the function
 // Use the string documentation to help you plan your solution
+
+function changeText(str) {
+  const capital = str.toUpperCase();
+  const underScore = "_";
+  return capital.split(" ").join(underScore);
```

Comment:
This is a correct answer ✅ ✅  
Another more succinct way could be to use the replaceAll function.